mix/ClientModel.cpp

```diff
@@ -173,6 +181,7 @@ void ClientModel::setupState(QVariantMap _state)
 		QString contractId = transaction.value("contractId").toString();
 		QString functionId = transaction.value("functionId").toString();
 		u256 gas = boost::get<u256>(qvariant_cast<QBigInt*>(transaction.value("gas"))->internalValue());
+		bool gasAuto = transaction.value("gas").toBool();
```

Comment:
"gas" mean two different things? should it be transaction.value("gasAuto") instead of transaction.value("gas") ?